src/test/java/net/sf/jabref/logic/bibtex/BibEntryWriterTest.java

```diff
@@ -62,6 +63,50 @@ public void testSerialization() throws IOException {
     }
 
     @Test
+    public void roundTripTestOtherType() throws IOException, URISyntaxException {
+        String bibtexEntry = "@Other{test," + OS.NEWLINE +
+                "  Comment                  = {testentry}" + OS.NEWLINE +
+                "}";
+
+        // read in bibtex string
+        ParserResult result = BibtexParser.parse(new StringReader(bibtexEntry), importFormatPreferences);
+        Collection<BibEntry> entries = result.getDatabase().getEntries();
+        BibEntry entry = entries.iterator().next();
+
+        String resourceName = "/testbib/othertype.bib";
```

Comment:
Please remove the assert statement here. Only one assert / failure reason per test. If you think that is valuable to test that the other type is written correctly, then extract this test to a new test method (I would actually prefer two tests for writing and reading separately instead of a roundtrip test)